sigridci/sigridci.py

```diff
@@ -31,6 +31,7 @@
 
 
 LOG_HISTORY = []
+SYSTEM_NAME_PATTERN = re.compile("[a-z0-9-]{2,}", re.IGNORECASE)
```

Comment:
A system name needs to be lower case. In addition, it cannot start with a dash, hence my suggestion.
```suggestion
SYSTEM_NAME_PATTERN = re.compile("[a-z0-9][a-z0-9-]+")
```

Reply:
@patveck We do a `.lower()` before using the customer and system name. I agree with the rest of the change, but if we disallow lowercase system names the script will break for peope that rely on the current implicit conversion to lowercase.